Spigot-Server-Patches/0594-Raytrace-entities-within-collection.patch

```diff
@@ -0,0 +1,24 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: NeumimTo <chleba48@gmail.com>
+Date: Wed, 28 Oct 2020 10:54:25 +0100
+Subject: [PATCH] Raytrace entities within collection
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/CraftWorld.java b/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
+index 299f57ca2a65887a0d7e7c584fc1bd5c783b0db2..9ee28222d8dbb3564d00e4ed18c4865ce5a62874 100644
+--- a/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
++++ b/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
+@@ -1304,6 +1304,13 @@ public class CraftWorld implements World {
+         BoundingBox aabb = BoundingBox.of(startPos, startPos).expandDirectional(dir).expand(raySize);
+         Collection<Entity> entities = this.getNearbyEntities(aabb, filter);
+ 
++        // Paper start - add entities collection
++        return rayTraceEntities(direction, maxDistance, raySize, startPos, entities);
```

Comment:
This is basically missing all of the sanity verifications which are done in the method above